### DISCUSSION TITLE
Table 'directus.directus_settings' doesn't exist

### DISCUSSION THREAD

**Discussion by Monte47** [directus/directus#2695] [Archive]
I am currently running a deployment of directus using kubernetes/docker. When I go to the index / sign in page I get "Something is wrong with this project":

`Statement could not be executed (42S02 - 1146 - Table 'directus.directus_settings' doesn't exist)`

I can't find anything in the documentation about a directus_settings table or any mandatory tables for that matter so I don't know what should be in there.

**Comment by omargohan** [comment]
Did you initialize the database? 

https://docs.directus.io/installation/docker.html#step-4-initialize-the-database-and-an-admin-user

**Reply by BassdoxXx** [reply]
Page not found...

**Comment by rijkvanzanten** [comment]
Looks like it's trying to run the migrations on a totally empty DB. Like @omargohan mentioned, make sure to initialize the DB when using Docker

**Comment by botzill** [comment]
Hi.

Why not running db populate automatically?

**Comment by tcrapts** [comment]
I have the same problem, I use the docker-compose.yml from the docs. I followed the docs, initialized the database and entered content. I moved the Directus folder to a different location under the impression that data would be persistent, now I get this error. It seems the database is not persistent. I lost quite some content for which I had no backup.

How can I make the database persistent?

**Reply by rijkvanzanten** [reply]
> I moved the Directus folder to a different location under the impression that data would be persistent, now I get this error.

The database is a MySQL database and isn't tied directly to the folder that holds the Directus files. You must've accidentally recreated the database or moved it somehow while (re)configuring the Docker setup.
